src/components/index.js

```diff
@@ -1,4 +1,3 @@
-import { Header } from "./Header";
-import { Footer } from "./Footer/Footer";
+import { Main } from "./Main/Main";
```

Comment:
- here you can directly export this ```Main``` component, no need to import it.
```suggestion
export { Main } from "./Main/Main";
```

Reply:
For thsi project I am following this approach, without any reason, sure I will use this approach in other projects.